index.ts

```diff
@@ -29,7 +29,7 @@ export const main = (): void => {
 
 try {
   // execute only if invoked as main script (rather than test)
-  if (require.main === module) main()
+  if (require.main === module) {main()}
```

Comment:
I think the issue @emdash-ie mentioned is coming from the use of `module` here, it's not available in esmm modules and it's not getting converted.

https://2ality.com/2022/07/nodejs-esm-main.html

Reply:
good call! I've tried this out as I'm working on some stuff for 10% time that requires actions-static-site. I made the changes in a branch and opened a PR into this branch: https://github.com/guardian/actions-static-site/pull/39

although I also had to make a change to the `tsconfig` in order to get `tsc` to work. I _think_ this was a pre-existing issue, but I'm not super confident with module resolution etc. in TS so would appreciate any perspectives on this 🙏